torch/csrc/jit/frontend/schema_type_parser.cpp

```diff
@@ -153,15 +153,16 @@ std::optional<AliasInfo> SchemaTypeParser::parseAliasAnnotation() {
       }
     }
     L.expect(')');
+    return alias_info;
   } else if (L.nextIf('!')) {
+    AliasInfo alias_info;
     alias_info.addBeforeSet(
         Symbol::fromQualString("alias::$" + std::to_string(next_id++)));
```

Comment:
Fmtlib could be used here.

Reply:
true, but outside the intended scope of the PR. I've updated the title to make that more clear.